port/unix/omrsysinfo.c

```diff
@@ -7783,7 +7783,53 @@ omrsysinfo_get_processes(struct OMRPortLibrary *portLibrary, OMRProcessInfoCallb
 	portLibrary->mem_free_memory(portLibrary, command);
 	closedir(dir);
 	return callback_result;
-#else /* defined(LINUX) */
+#elif defined(OSX) /* defined(LINUX) */
+	uintptr_t callback_result = 0;
```

Comment:
- Variable names should follow camel case as per the [OMR coding standard](https://github.com/eclipse-omr/omr/blob/3223d8d0d584575eafb5ec462250baae92cd6084/doc/CodingStandard.md#variables).
- I also noticed that two of your earlier PRs (https://github.com/eclipse-omr/omr/pull/7681 and https://github.com/eclipse-omr/omr/pull/7772) use inconsistent naming that does not fully adhere to this convention. It would be good to open a follow-up PR to align those variable names with the standard.